baus/datasources.py

```diff
@@ -256,39 +256,38 @@ def reprocess_dev_projects(df):
     # record, we change the later ones to add records - we don't want to
     # constantly be redeveloping projects, but it's a common error for users
     # to make in their development project configuration
-    df = df.sort_values(["geom_id", "year_built"])
-    prev_geom_id = None
+    df = df.sort_values(["parcel_id", "year_built"])
+    prev_pcl_id = None
     for index, rec in df.iterrows():
-        if rec.geom_id == prev_geom_id:
+        if rec.parcel_id == prev_pcl_id:
             df.loc[index, "action"] = "add"
-        prev_geom_id = rec.geom_id
+        prev_pcl_id = rec.parcel_id
 
     return df
 
 
 # shared between demolish and build tables below
 def get_dev_projects_table(scenario, parcels):
-    return pd.DataFrame()
-    df = pd.read_csv(os.path.join(misc.data_dir(), "development_projects.csv"))
+    df = pd.read_csv(os.path.join(misc.data_dir(),
+                     "development_projects_w_apn.csv"))
+    # find nearest parcel centroid to dev project x, y and assign parcel_id
+    df = pd.merge(df, parcels.apn.reset_index(), how='left', left_on='apn',
+             right_on='apn')
```

Comment:
I like this merge.  Clever way to merge to a series - I hadn't seen that before.